addons/maintenance/tests/test_calendar_with_recurrence.py

```diff
@@ -6,75 +6,63 @@
 @tagged('post_install', '-at_install')
 class TestCalendarWithRecurrence(HttpCase):
 
-    def test_dblclick_event_from_calendar(self):
-        self.env['maintenance.team'].create({
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.today_as_weekday = date.today().weekday() + 1  # starts at 1 instead of 0, useful in multiple occasions
+        cls.today_at_midnight = datetime.combine(date.today(), datetime.min.time())
+        cls.team = cls.env['maintenance.team'].create({
             'name': 'the boys',
         })
-        equipment = self.env['maintenance.equipment'].create({
+        cls.equipment = cls.env['maintenance.equipment'].create({
             'name': 'room'
         })
-        self.env['maintenance.request'].create({
+        cls.past_request = cls.env['maintenance.request'].create({
             'name': 'send the mails',
             'schedule_date': datetime.today() - timedelta(weeks=2),
         })
-        request = self.env['maintenance.request'].create({
+        cls.present_request = cls.env['maintenance.request'].create({
             'name': 'clean the room',
-            'schedule_date': datetime.combine(date.today(), (datetime.min + timedelta(hours=10)).time()),  # today at 10.00 AM
-            'equipment_id': equipment.id,  # necessary for the tour to work with mrp_maintenance installed
+            'schedule_date': cls.today_at_midnight + timedelta(hours=10),  # today at 10.00 AM
+            'equipment_id': cls.equipment.id,  # necessary for the tour to work with mrp_maintenance installed
             'maintenance_type': 'preventive',
             'recurring_maintenance': True,
             'repeat_interval': 1,
             'repeat_unit': 'day',
             'duration': 1,
         })
-        self.env['maintenance.request'].create({
+        cls.future_request = cls.env['maintenance.request'].create({
             'name': 'wash the car',
             'schedule_date': datetime.today() + timedelta(weeks=1),
         })
+        cls.request_calendar_action = cls.env["ir.actions.actions"]._for_xml_id("maintenance.hr_equipment_request_action_cal")
+        cls.request_calendar_url = '/web?#action=%s' % (cls.request_calendar_action['id'])
+
+    def test_dblclick_event_from_calendar(self):
+        timelang = self.env['res.lang'].create({
+            'name': 'timelang',
+            'code': 'en_tl',
+            'url_code': 'en_tl',
+            'week_start': str(self.today_as_weekday),
```

Comment:
Seems a bit hacky...

Isn't simpler to create 8 occurrences, and, on the tour:
\- click on the next week
\- select the first occurrence

I don't know if it would still test the issue ?

By the way, can you add a docstring to both tests ? So we can quickly understand what they are about (it's actually better to always add such a description)